### ISSUE TITLE
scroll relative to cursor

### ISSUE THREAD

**Issue by ri-aje** [ranger/ranger#2047]
#### Runtime Environment

- Operating system and version: 
- Terminal emulator and version: 
- Python version: 
- Ranger version/commit: 
ranger version: ranger-master v1.9.3-103-gfdbdb797
Python version: 3.7.1 (default, Sep 21 2019, 10:53:58) [GCC 7.4.0]
Locale: en_US.UTF-8
- Locale: 

#### Current Behavior
N/A

#### Expected Behavior
`z[enter]/zz/zb` to scroll the view window relative to the cursor, much like the same shortcuts do in vim. don't think this is available today.

#### Context

sometimes need to see more files when cursor is close to the bottom line, need a way to bring it back to the middle of the screen, without positioning the cursor on a different file.

`scroll_offset` helps and works most of the time, but occasionally I need more context around a file, and these new additions will do better.

#### Possible Solutions
probably need a new function to scroll the underlying view while keeping the cursor in position.

#### Steps to reproduce
1. 
2. 
3. 

#### Traceback

```

```

**Comment by NoSuck** [comment]
I agree this is a good idea and probably one of those things that I have been working around manually for years without even consciously thinking.
